### ISSUE TITLE
'compare' is deprecated.

### ISSUE THREAD

**Issue by jjandshi** [jean997/cause#8]
Has anyone encountered below problem, when practicing "cause" example? And how could i solve it? Thanks,

> res <- cause(X=X, variants = top_vars, param_ests = params)
Estimating CAUSE posteriors using  1215  variants.
Warning message:
'compare' is deprecated.
Use 'loo_compare' instead.
See help("Deprecated")

**Comment by jean997** [comment]
Yes this is a new warning since an update to the `loo` package. Currently everything still works and it is just a warning. I need to go in and update the function call -- coming soon! For now don't worry about it and in a future version you will stop seeing the warning.
